vortex-array/src/vtable/statistics.rs

```diff
@@ -22,3 +23,35 @@ where
         StatisticsVTable::compute_statistics(encoding, array_ref, stat)
     }
 }
+
+pub fn compute_statistics(array: &Array, stat: Stat) -> VortexResult<StatsSet> {
```

Comment:
For most other non-compute vtables, this is wrapped in an `impl Array { ... }` block